docs/README.md

```diff
@@ -36,9 +36,9 @@
 - [SAT Kibana Dashboards](dashboards/SAT_Kibana_Dashboards.md)
 - [SAT Grafana Dashboards](dashboards/SAT_Grafana_Dashboards.md)
 
-## [SAT Usage](usage.md)
+## SAT Usage
 
-- [SAT Bootprep](usage.md#sat-bootprep)
+- [SAT Bootprep](sat_bootprep.md)
```

Comment:
I'd suggest removing lines 39 and 40 and having line 41 be its own level 2 heading. It doesn't really make sense to have a single topic under "SAT Usage" right now. However, I'd be all for adding the heading back if we have more topics dealing with usage besides SAT bootprep in the future.

Reply:
What are you envisioning the content would be under this level-2 heading in that case? I think a level-2 heading with no content would look a bit odd, so are you proposing that I add links into the specific sections of that document? Something like this?

```
- [SAT bootprep vs SAT bootsys](sat_bootprep.md#sat-bootprep-vs-sat-bootsys)
- [Editing a bootprep input file](sat_bootprep.md#editing-a-bootprep-input-file)
- [Example bootprep input files](sat_bootprep.md#example-bootprep-input-files)
- [Viewing built-in generated documentation](sat_bootprep.md#viewing-built-in-generated-documentation)
```

It seems like it's maybe exposing too much detail at this top-level of the SAT docs. Thoughts?

Reply:
I pushed the change described in my comment above. Let me know what you think.

Reply:
These changes are exposing a level of detail, but I think that might actually be a good thing. My understanding is that SAT bootprep is pretty important to the install and upgrade process, and there were quite a few changes we made to this information for Papaya. This might help users understand and better navigate what's in this section when they first come to the docs.

Reply:
Sounds good. I'll leave it.